.changeset/shiny-beds-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': patch
```

Comment:
Should this be a major version since this will cause breaking changes that need to be addressed where these partials are used in the Wordpress site?

Reply:
Good catch, thanks! 👍

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1286/commits/94dfbd6be403ff7d2952a81e2aa78eabfdeb44f0